addons/l10n_id_efaktur/tests/test_l10n_id_efaktur.py

```diff
@@ -0,0 +1,60 @@
+from odoo.tests import tagged, common
+from odoo.tools import float_round
+from odoo.addons.l10n_id_efaktur.models.account_move import FK_HEAD_LIST, LT_HEAD_LIST, OF_HEAD_LIST, _csv_row
+
+@tagged('post_install', '-at_install')
+class TestIndonesianEfaktur(common.TransactionCase):
+    def setUp(self):
+        """
+        1) contact with l10n_id_pkp=True, l10n_id_kode_transaksi="01"
+        2) tax: amount=10, type_tax_use=sale, price_include=True
+        3) invoice with partner_id=contact, journal=customer invoices,
+        """
+        super().setUp()
+
+        # change company info for csv detai later
+        self.env.company.country_id = self.env.ref('base.id')
+        self.env.company.street = "test"
+        self.env.company.phone = "12345"
+
+        self.partner_id = self.env['res.partner'].create({"name": "l10ntest", "l10n_id_pkp": True, "l10n_id_kode_transaksi": "01", "l10n_id_nik": "12345"})
+        self.tax_id = self.env['account.tax'].create({"name": "test tax", "type_tax_use": "sale", "amount": 10.0, "price_include": True})
+
+        self.efaktur = self.env['l10n_id_efaktur.efaktur.range'].create({'min': '0000000000001', 'max': '0000000000010'})
+        self.out_invoice_1 = self.env['account.move'].create({
+            'type': 'out_invoice',
+            'partner_id': self.partner_id.id,
+            'invoice_date': '2019-05-01',
+            'date': '2019-05-01',
+            'invoice_line_ids': [
+                (0, 0, {'name': 'line1', 'price_unit': 110.0, 'tax_ids': self.tax_id.ids}),
+            ],
+            'l10n_id_kode_transaksi': "01",
+        })
+        self.out_invoice_1.post()
+
+    def _efaktur_csv(self, move):
+        """constructing the output_head string like the one generated """
+
+        output_head = '%s%s%s' % (
+            _csv_row(FK_HEAD_LIST, ','),
+            _csv_row(LT_HEAD_LIST, ','),
+            _csv_row(OF_HEAD_LIST, ','),
+        )
+
+        # remaining lines
+        line_4 = '"FK","01","0","0000000000001","5","2019","1/5/2019","12345","l10ntest","","100","10","0","","0","110","0","0","INV/2019/0001 12345","0"\n'
+        line_5 = '"FAPR","000000000000000","YourCompany","test","","","","","","","","","","12345"\n'
+
+        # with previous iteration, the unit price included will be 110 and then the discount will be 10 which is an error
+        line_6 = '"OF","","","100","1.0","100","0","100","10","0","0"\n'
+        return output_head + line_4 + line_5 + line_6
+
+    def test_efaktur_csv_output(self):
```

Comment:
Give this test a number and a more specific name explaining what you are testing